frontend/src/components/header/MenuOverlay.js

```diff
@@ -548,19 +582,24 @@ class MenuOverlay extends Component {
                 <div className="input-flex input-primary">
                   <i className="input-icon meta-icon-preview" />
 
-                  <DebounceInput
-                    debounceTimeout={250}
+                  <input
                     type="text"
-                    inputRef={(ref) => {
+                    ref={(ref) => {
```

Comment:
Can we move this to a `setRef` function, so that we won't have an anonymous function here for no reason ?

Reply:
Done